src/posts/2022-02-10-design-het-web.md

```diff
@@ -0,0 +1,68 @@
+---
+title: 'Design het Web'
+category: inspiration
+tags: new blogpost
+comments: true
+author: joris
+min_read: 5
+date: 2022-02-10
+permalink: /2022/02/10/design-het-web/
+---
+
+<!-- Section: Intro -->
+
+Development en webdesign zijn termen die vaak door elkaar worden gehaald. Hoewel het twee aparte taken zijn varen ze wel in hetzelfde vaarwater en zullen ze in een groot deel van de web-trajecten overlappen. Uit mijn ervaring blijkt dat veel personen die in één van deze twee categorieën werken ook behendig zijn in de ander, maar hebben altijd voorkeur voor hun eigen onderdeel. Welke mogelijkheden zijn er voor wie als we het hebben over doorgroeien(als designer)?
+
+Er zijn een aantal mogelijkheden waar je uit kan kiezen als designer: je wil doorleren en de code echt gaan beheersen, je wilt omscholen en het design achter je laten, je vindt allebei leuk en blijft er tussenin hangen of je sluit de code helemaal uit.
```

Comment:
>Er zijn een aantal mogelijkheden waar je uit kan kiezen als designer:

After this, I would expect a list to sum-up the options.
- e.g. one
- e.g. two

> je wil doorleren
...
je wilt omscholen

Choose one or the other 😉 (wil/wilt)

>tussenin hangen, of je

Personally I would expect a comma (,) there. But I ain't a dutch grammer pro myself 🙂